docs/en-us/manual/introduction.md

```diff
@@ -12,24 +12,24 @@ icon: mdi:information-outline
     Make sure the screen stays on the stage detail page with the **Start** and **Auto-Deploy** buttons available.
 - If you are not on this page, `Current/Previous` will automatically navigate to the last stage played according to the record in the lower right corner of the terminal homepage.
 - You can also enable `Manually input stage name` in `Task Settings` - `Combat` - `Advanced` and enter the stage number manually. Currently supported stages include:
-    - All main theme stages, where `-NORMAL` or `-HARD` can be added at the end to switch between standard and challenge modes.
-    - LMD stages and Battle Record stages 5/6. The input should be `CE-6` or `LS-6` even if you have not unlocked it yet. In that case, the program will automatically switch to corresponding stage 5.
-    - Skill Summary, Shop Voucher, and Carbon Stages 5. The input also should be `CA-5`, `AP-5`, and `SK-5` respectively.
-    - Chip stages. The input should be complete with stage number, such as `PR-A-1`.
-    - Annihilation. The input should be `Annihilation`.
-    - Certain side story stages. This includes `OF-1`, `OF-F3` and `GT-5`.
-    - The last three stages of the current SS event. This is available after downloading updates automatically from the [API](https://ota.maa.plus/MaaAssistantArknights/api/gui/StageActivity.json) when the event is on. Prompt will be shown in the main page when this is available.
-    - For the SS event rerun, you can enter `SSReopen-XX` to clear XX-1 ~ XX-9 levels once. Example `SSReopen-IC`.
+  - All main theme stages, where `-NORMAL` or `-HARD` can be added at the end to switch between standard and challenge modes.
+  - LMD stages and Battle Record stages 5/6. The input should be `CE-6` or `LS-6` even if you have not unlocked it yet. In that case, the program will automatically switch to corresponding stage 5.
+  - Skill Summary, Shop Voucher, and Carbon Stages 5. The input also should be `CA-5`, `AP-5`, and `SK-5` respectively.
+  - Chip stages. The input should be complete with stage number, such as `PR-A-1`.
+  - Annihilation. The input should be `Annihilation`.
+  - Certain side story stages. This includes `OF-1`, `OF-F3` and `GT-5`.
+  - The last three stages of the current SS event. This is available after downloading updates automatically from the [API](https://ota.maa.plus/MaaAssistantArknights/api/gui/StageActivity.json) when the event is on. Prompt will be shown in the main page when this is available.
+  - For the SS event rerun, you can enter `SSReopen-XX` to clear XX-1 ~ XX-9 levels once. Example `SSReopen-IC`.
```

Comment:
All of the indentations are not consistent! Again!
I highly suggest using 4 space indentation. I fixed all of them this morning!

Reply:
Oh really?! I wasn't using markdownlint... But then it doesn't make sense, some original zh-cn files are indented with 2 and some with 4...
Strange. I'll check it out!

Reply:
well, actually yes
an example here:
2 and 4

![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/55343783/a8fa5a72-321e-42f5-9354-95fe53465dd2)
all 2

![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/55343783/709ecda0-d1fa-474f-8c26-6298e9f52741)

the [doc](https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md007.md) markdownlint gave

Reply:
I installed markdownlint and it seems to be working because if I change a list removing a '.' it will warn me, but it doesn't warn me for spaces / tabs.

I'm also trying to use Prettier to auto format everything and the consistency goes completely out of the window:
2 spaces, 3 spaces and 4 spaces for different things... To be honest it's a bit too much, because everything has a different indentation and vscode really doesn't like it as it has no idea where to draw the lines
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/56174894/c4917e05-d5b7-42f6-b637-5f2aad19447d)

Reply:
check this? https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/Prettier.md

Reply:
Could you do me a favor and try installing Prettier and formatting everything with the out of the box settings?
As it seems my markdownlint is not working correctly. I want to make sure prettier is doing the correct formatting.

Reply:
~~i just noticed that i had installed prettier but never used~~
so it removed **one** space here
i've got no idea now
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/55343783/94edfd81-ed58-4bfe-b9f2-ca05b7fd9cf2)